### ISSUE TITLE
Move docs site and web site to github.com

### ISSUE THREAD

**Issue by larstobi** [ddccontrol/ddccontrol#15]
It still points to http://ddccontrol.sourceforge.net and we want it to point to github.

Make a new web site, or try to copy the one at sf.net and make one here.

**Comment by kravemir** [comment]
[GitHub pages](https://pages.github.com/) and [Read the Docs](https://readthedocs.org/) could be used for such purpose.

**Comment by larstobi** [comment]
I agree! I thought we could just put everyging in GitHub Pages, but putting the docs on Read the Docs is a much better idea. 👍

**Comment by anarcat** [comment]
well RTD would be really nice, but unfortunately it doesn't support docbook as a formatter (as far as I know) - only markdown and RST/Sphinx. So a conversion would be required for RTD. It's probably possible to do something like that with Pandoc and some hand-crafting, and I think would be easier to maintain in the long term. But it's not a short term fix, and I think such a fix is needed quickly.

GH pages, however, support arbitrary HTML / webpages, provided they are rendered on a branch. I would recommend this be done in a distinct repo to avoid cluttering the main repository with build artefacts. This would be in a ddccontrol.github.io repository or something like that. That could be an easy short term fix, and I would recommed that as a first step.

**Comment by kravemir** [comment]
What about domain? I think, it would be good to have permanent domain for the project, which isn't tied to (git) hosting provider.

For example, project could become part of some free software organization. For example, freedesktop.org.

Having permanent custom domain for project helps with unique identification of the project. For example, it would be useful for D-Bus interface name: https://dbus.freedesktop.org/doc/dbus-specification.html#message-protocol-names-interface

> Interface names should start with the reversed DNS domain name of the author of the interface (in lower-case), like interface names in Java...

**Comment by anarcat** [comment]
sure, a domain sounds fine. just be careful it doesn't expire if you rent it yourself...

**Comment by larstobi** [comment]
The website has been moved to Github Pages at https://ddccontrol.github.io
